storm-client/src/jvm/org/apache/storm/topology/WindowedBoltExecutor.java

```diff
@@ -205,10 +205,21 @@ protected void validate(Map<String, Object> topoConf, Count windowLengthCount, D
         }
         // validate
         validate(topoConf, windowLengthCount, windowLengthDuration,
-                 slidingIntervalCount, slidingIntervalDuration);
-        evictionPolicy = getEvictionPolicy(windowLengthCount, windowLengthDuration);
-        triggerPolicy = getTriggerPolicy(slidingIntervalCount, slidingIntervalDuration,
-                                         manager, evictionPolicy);
+            slidingIntervalCount, slidingIntervalDuration);
+        if (topoConf.containsKey(Config.TOPOLOGY_BOLTS_WINDOW_EVICTION_POLICY)
+            && topoConf.get(Config.TOPOLOGY_BOLTS_WINDOW_EVICTION_POLICY) instanceof EvictionPolicy) {
```

Comment:
I don't think you need to do the instanceof check here, we already specify that it needs to be that type in Config.